error.go

```diff
@@ -0,0 +1,16 @@
+package kubeconfig
+
+import "github.com/giantswarm/microerror"
+
+var invalidConfigError = &microerror.Error{
+	Kind: "invalidConfigError",
+}
+
+var missingKubeConfigError = &microerror.Error{
```

Comment:
```suggestion
var notFoundError = &microerror.Error{
```

We usually use this naming.